docs/use-mainnet/fund.mdx

```diff
@@ -142,24 +167,64 @@ If you already have tokens on another network, you can [follow the instructions
 
 ## Steps to buy tokens on Linea
 
-1. Navigate to [portfolio.metamask.io](https://portfolio.metamask.io), and connect the account to which you want your purchased tokens to be delivered.
-2. If you're on a mobile device, select 'Buy' from the hamburger icon menu located in the top left of the screen; on desktop, select the 'Buy' tab on the left.
-3. Now you'll see the Buy interface. If you've never used it before, you'll first need to **select your region**.
+1. Navigate to [portfolio.metamask.io](https://portfolio.metamask.io), and connect the account to 
+which you want your purchased tokens to be delivered.
+2. If you're on a mobile device, select 'Buy' from the hamburger icon menu located in the top left 
+ of the screen; on desktop, select the 'Buy' tab on the left.
+3. Now you'll see the Buy interface. If you've never used it before, you'll first need to 
+**select your region**.
+   This information is required because availability of payment methods and providers vary based on 
+   your bank's location.
+4. **Set the payment method**. Again, if you've used Buy before on your current device, your 
+previous choice should be loaded by default.
+5. **Select a token**. The token selection menu also allows you to switch networks; it's here 
+where you need to locate and select Linea from the networks list.
+6. Now you'll see the tokens available to buy on Linea. Pick the one you want to buy, then input 
+the amount on the main screen once the tokens menu is gone.
+7. You'll instantly see one or more quotes on the right, if you're on a desktop browser. On mobile 
+devices, hit 'Get Quotes' to see them.
+8. Pick a quote by clicking on it.
+
+Selecting a quote redirects you to the provider's site to complete the transaction. You may need 
+to complete KYC ("Know Your Customer"; essentially, identifying yourself) with the provider.
+
+After you've completed the steps with the provider, you'll be routed back to MetaMask Portfolio. 
+Timescales for receiving your funds will vary based on the provider and token route.
 
-   This information is required because availability of payment methods and providers vary based on your bank's location.
+</TabItem>
 
-4. **Set the payment method**. Again, if you've used Buy before on your current device, your previous choice should be loaded by default.
+<TabItem value="Linea Sepolia">
 
-5. **Select a token**. The token selection menu also allows you to switch networks; it's here where you need to locate and select Linea from the networks list.
+## Linea Sepolia faucets
 
-6. Now you'll see the tokens available to buy on Linea. Pick the one you want to buy, then input the amount on the main screen once the tokens menu is gone.
+An up-to-date list of available Linea Sepolia faucets is available [here](../build-on-linea/goerli-to-sepolia.mdx#faucets).
```

Comment:
Is there a reason we are redirecting to the other page rather than putting it directly on this page

Reply:
No reason other than it'll prevent us having to update the information in two places. The faucets aren't all currently live so the information will need updating in `goerli-to-sepolia` at some point. Appreciate though that being sent round the docs isn't ideal UX.